TodayExtension/Base.lproj/MainInterface.storyboard

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13196" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="M4Y-Lb-cyx">
-    <device id="retina4_7" orientation="portrait">
-        <adaptation id="fullscreen"/>
-    </device>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="18122" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="M4Y-Lb-cyx">
```

Comment:
Not sure if any adding Turkish should affect the Stpryboards at all.  Sorry for being cast. obvious but you could have accidentally `touched` the storyboard and that would register a change.  Will you check , unstage and / or explain my confusion?

Reply:
Not sure either. I know they changed when I first added localization because I had to unckeck the boxes on the storyboards... reverted. re-review requested